pom.xml

```diff
@@ -64,6 +64,43 @@
     <github.global.server>github</github.global.server>
     <site.installationModule>google-cloud-spanner-parent</site.installationModule>
     <google.cloud.shared-dependencies.version>0.18.0</google.cloud.shared-dependencies.version>
+
+    <!-- finalized versions -->
+    <!-- JRE for Java 8. Beam's dependency (gcsio) uses JRE flavor -->
+    <guava.version>30.1.1-jre</guava.version>
+    <google.autovalue.version>1.8.1</google.autovalue.version>
+    <protobuf.version>3.16.0</protobuf.version>
+    <http.version>1.39.2-sp.1</http.version>
+    <io.grpc.version>1.36.2</io.grpc.version>
+    <!-- We don't use gax-bom because it includes the artifacts with 'testlib' classifier.
+        When updating gax.version, update gax.httpjson.version too. -->
+    <gax.version>1.64.0-sp.1</gax.version>
+    <gax.httpjson.version>0.81.0-sp.1</gax.httpjson.version>
+    <google.api.client.version>1.31.3-sp.1</google.api.client.version>
+    <api.common.version>1.10.1-sp.1</api.common.version>
+    <google.auth.library.version>0.25.2-sp.1</google.auth.library.version>
+    <google.oauth.client.version>1.31.4-sp.1</google.oauth.client.version>
+    <google.cloud.spanner.version>3.3.3-sp.1</google.cloud.spanner.version>
+    <google.cloud.monitoring.version>2.1.0-sp.1</google.cloud.monitoring.version>
+    <google.cloud.trace.version>1.3.0-sp.1</google.cloud.trace.version>
+    <google.cloud.pubsub.version>1.111.0-sp.1</google.cloud.pubsub.version>
+    <google.cloud.bigquery.version>1.127.12-sp.1</google.cloud.bigquery.version>
+    <google.cloud.bigtable.version>1.22.0-sp.1</google.cloud.bigtable.version>
+    <bigtable-hbase-beam.version>1.20.0-sp.1</bigtable-hbase-beam.version>
+    <google.cloud.storage.version>1.113.14-sp.1</google.cloud.storage.version>
+    <datastore.v1.proto.client.version>1.6.4-sp.1</datastore.v1.proto.client.version>
+    <appengine.api.1.0.sdk.version>1.9.86</appengine.api.1.0.sdk.version>
+    <androidpublisher.version>v3-rev20210429-1.31.0</androidpublisher.version>
+
```

Comment:
The dependencies below this line are the ones we had to "guess" and fix based on errors of the maven build enforcer checks (for transitive versions compatibilities). These would usually come from the google-cloud-shared-libraries, but we did not know which version of those to use.

@elharo Is this the approach we would like to take or will we release a version of the google-cloud-shared-libraries that is compatible with the lts versions of all the libraries we have?